metadata/preview/azure-ai-vision-common-internal.json

```diff
@@ -0,0 +1,11 @@
+{
+    "Name": "azure-ai-vision-common-internal",
```

Comment:
Forgot to comment earlier that, azure-ai-vision-common-internal is internal package and would not be required in reference API documentation.

Please remove this from the docs.

Reply:
We're keeping this one.